saleor/checkout/utils.py

```diff
@@ -645,26 +652,30 @@ def change_shipping_address_in_checkout(checkout, address):
         checkout.save(update_fields=["shipping_address"])
 
 
-def get_checkout_context(
-    checkout, discounts, taxes, currency=None, shipping_range=None
-):
+def get_checkout_context(checkout, discounts, currency=None, shipping_range=None):
     """Data shared between views in checkout process."""
-    checkout_total = checkout.get_total(discounts, taxes)
+    checkout_total = (
+        calculate_checkout_total(checkout=checkout, discounts=discounts)
+        - checkout.get_total_gift_cards_balance()
+    )
+    checkout_total = max(checkout_total, ZERO_TAXED_MONEY)
+    checkout_subtotal = calculate_checkout_subtotal(checkout, discounts)
+    shipping_price = calculate_checkout_shipping(checkout, discounts)
+
     shipping_required = checkout.is_shipping_required()
-    checkout_subtotal = checkout.get_subtotal(discounts, taxes)
     total_with_shipping = TaxedMoneyRange(
         start=checkout_subtotal, stop=checkout_subtotal
     )
     if shipping_required and shipping_range:
-        total_with_shipping = shipping_range + checkout_subtotal
+        total_with_shipping = shipping_range + checkout_subtotal.net
 
     context = {
         "checkout": checkout,
-        "checkout_are_taxes_handled": bool(taxes),
+        "checkout_are_taxes_handled": checkout_are_taxes_handled(),
```

Comment:
Sounds to me like "taxes were applied to the checkout" where it's actually "taxes are enabled". Maybe we should refactor the name?